### PR TITLE
Pin the base image including security patches

### PR DESCRIPTION
**Pull request by Annegies**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM alpine
+FROM alpine:3.11.5
```
Comment (patrick-east) [inline Dockerfile]: I suspect we just need to cut a new release with an updated base image. These are effectively the same (alpine:latest == alpine:3.11.5)

I'm not even sure if kube-mgmt needs anything from the OS.. maybe we just switch to distroless? @tsandall thoughts?

Comment (tsandall) [reply Dockerfile:1]: A bunch of users are relying on the kube-mgmt image to generate the authz policy for OPA itself. This has been embedded into the Helm chart: https://github.com/helm/charts/blob/master/stable/opa/templates/deployment.yaml#L38

If we switch to distroless, will that break?

**Comment by jchauncey** [comment]
Have you considered moving to the debian-base image that the other kubernetes projects use? I am analyzing base images used by the various projects we deploy here at Microsoft and would love to see some consistency which would help with vulnerability scans and patching. There is a new image here - `us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:v2.1.0` and I am goign to work with sig-release to help keep this image up to date so other projects can rely on it.

**Comment by tsandall** [comment]
@jchauncey that sounds useful. @patrick-east what are your thoughts? I'd be inclined to use the same base image that k8s sig-release recommends.

**Comment by patrick-east** [comment]
+1 for using a recommended (and maintained) base image

**Comment by tsandall** [comment]
Closing this PR as it's been open for a while without any updates.

**Pull request closed without merging**